index.html

```diff
@@ -273,7 +284,27 @@
 Privacy therefore depends on having an equivalently strong countervailing force.
 
 
-## Exclusion
+### Age Verification {#age}
+
+A number of governments have chosen to mandate the use of age verification
+for access to certain online services.
+The TAG is not confident that age verification is even appropriate ([[SUPPRESS]]),
+but we are waiting for the outcome of the upcoming [[AGE-WS]] workshop
+before taking a firm position.
+
+Different jurisdictions vary in how they require age verification,
+but a common theme is to place the responsibility on the service provider.
```

Comment:
```suggestion
but a common theme is to place the responsibility on the service provider
(the website, in the web's context).
```

Since browsers and issuers also provide services, right?